src/transport/SecureMessageCodec.cpp

```diff
@@ -48,6 +48,16 @@ CHIP_ERROR Encrypt(const CryptoContext & context, CryptoContext::ConstNonceView
 
     ReturnErrorOnFailure(payloadHeader.EncodeBeforeData(msgBuf));
 
+// Skip encryption and message integrity!
+#if CHIP_CONFIG_SECURITY_FUZZ_MODE
+#warning                                                                                                                           \
+    "Warning: CHIP_CONFIG_SECURITY_FUZZ_MODE=1 bypassing encryption! Node can only communicate with other nodes built with this flag set. Requires build flag 'treat_warnings_as_errors=false'."
```

Comment:
The "requires build flag" bit is:
1. Not really in need of being documented here.
2. Seems to not be needed at all, if this code just has `#else` around the bits we want to turn off....

Same for Decrypt.